src/Service/ProducerEventProcessorTest.cpp

```diff
@@ -1610,4 +1615,54 @@ TEST(ProducerEventProcessor, MetadataEvent) {
   EXPECT_EQ(actual_metadata_event.info_event().message(), kMessage);
 }
 
+TEST(ProducerEventProcessor, MemoryEventWrapper) {
+  MockCaptureEventBuffer buffer;
+  auto producer_event_processor = ProducerEventProcessor::Create(&buffer);
```

Comment:
Nit: Could you move these two lines down to where `buffer` is used first time?

Reply:
Done! Thanks!